core/integration/features/verified/0034-PROB-liquidity_TauScaling.feature

```diff
@@ -0,0 +1,299 @@
+Feature: Tests impact from change of tau.scaling parameter on probability of trading, 0038-OLIQ-006
+
+  Scenario: 001: set tau.scaling to 1
+  Background:
+    Given the following network parameters are set:
```

Comment:
let's explicitly state the LP range and also somewhere print what the price monitoring bounds are or show where on the book the LP volume is actually landing